extractor/text_const.go

```diff
@@ -0,0 +1,86 @@
+/*
+ * This file is subject to the terms and conditions defined in
+ * file 'LICENSE.md', which is part of this source code package.
+ */
+
+package extractor
+
+// The follow constant configure debugging.
```

Comment:
"The following constants"
Is this something we want to keep to help with debugging in development going forward?  Seems pretty clean, good to have constants

Reply:
I may need the constants for table, form and invoice extraction development.
A large fraction of my development time is spent looking at PDF marking operators, text mark posiitions, word tokenization, word proximity and cell structures.  Each of these individually is many log lines and I find it hard to read them when they are mixed together. The constants allow me to look at one at a time.
We can delete them when I am done that development.

Reply:
Fine to keep them too I think.  Clean and no effect on performance.